realtime_ai_character/llm/base.py

```diff
@@ -137,7 +137,8 @@ def question(self, query: str, apiKey: str, brainId: str) -> str:
                 "model": "gpt-3.5-turbo-16k",
                 "temperature": 0.5,
                 "question": query,
-                "max_tokens": 512,
+                "max_tokens": 256,
```

Comment:
why lower?

Reply:
moved to default value to test if it will not 500, but unfortunately the production quivr service is still not operating on this endpoint